src/lib/transition_frontier/persistent_frontier/database.mli

```diff
@@ -47,7 +47,7 @@ val close : t -> unit
 val check :
      t
   -> genesis_state_hash:State_hash.t
-  -> ( unit
+  -> ( State_hash.t
```

Comment:
looks like this is ignored at the call site?

Reply:
There's 2 call sites. One of them would ignore this return value; another one (https://github.com/MinaProtocol/mina/blob/3317fb2d041566a433118b08a4a9396d7a0d04da/src/lib/transition_frontier/persistent_frontier/persistent_frontier.ml#L155) would use it to pick the right snarked_ledger from checkpoints.